Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -190,6 +191,9 @@ With the block uncommented, you should see a blue ribbon appear at the top of th
 
 Gradle will use the configuration settings we just activated to install all of the Edge Bridge extensions we want for our tutorial app, and allow us to use their features within the app's code.
 
+> **Note**  
+> After this step, there are no Adobe Experience Cloud Solution extensions in the app. At this point, the import and registration for the AEPIdentity extension can also be removed since AEPEdgeIdentity takes care of the identity functionality for the Edge extensions.
+> If your application still uses Adobe Experience Cloud Solution extensions, such as Adobe Target, Adobe Campaign, etc. (find the full list [here](https://aep-sdks.gitbook.io/docs/)) you should ignore the steps below for removing AEPIdentity and continue to use the extension.
```

Comment:
Some renaming for Android:
AEPIdentity -> Identity
AEPEdgeIdentity -> Identity for Edge Network

Reply:
nit: one more at line 196